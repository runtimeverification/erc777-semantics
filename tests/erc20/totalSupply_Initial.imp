test.preamble();

// Testing totalSupply() on initial configuration

test.totalSupply();
