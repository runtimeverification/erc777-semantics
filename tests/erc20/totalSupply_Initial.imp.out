Total supply is 550
